odl/contrib/fom/supervised.py

```diff
@@ -213,61 +193,51 @@ def standard_deviation_difference(data, ground_truth, mask=None,
                                   normalized=False):
     """Return absolute difference in std between ``data`` and ``ground_truth``.
 
-    Evaluates difference in standard deviation (std) between input (``data``)
-    and reference (``ground_truth``). Allows for normalization (``normalized``)
-    and a masking of the two spaces (``mask``).
+    Parameters
+    ----------
+    data : `FnBaseVector`
+        Input data to compare to the ground truth.
+    ground_truth : `FnBaseVector`
+        Reference to compare ``data`` to.
+    mask : `array-like`, optional
+        If given, ``data * mask`` is compared to ``ground_truth * mask``.
+    normalized  : bool, optional
+        Boolean flag to switch between unormalized and normalized FOM.
+
+    Returns
+    -------
+    sdd : float
+        FOM value, with higher value meaning lower correspondence.
```

Comment:
I would overall change "correspondence" to something like "meaning a better fit", hard to read as is